astropy/coordinates/baseframe.py

```diff
@@ -508,9 +509,9 @@ def __init__(self, *args, copy=True, representation_type=None,
                     # come from the representation instead of the frame's
                     # attribute names.
                     try:
-                        representation_data = representation_cls._unit_representation(copy=copy,
-                                                                 **repr_kwargs)
-                    except Exception as e2:
+                        representation_data = representation_cls\
```

Comment:
That slash isn't encouraged either. Maybe `# noqa` is cleaner here, or trying breaking the line after `(`.

Reply:
You can even do
```
(representation_cls
 ._unit...
```